main/main.go

```diff
@@ -25,13 +25,20 @@ import (
 	"github.com/ava-labs/awm-relayer/vms/vmtypes"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/spf13/pflag"
 	"go.uber.org/atomic"
 	"go.uber.org/zap"
 	"golang.org/x/sync/errgroup"
 )
 
+var version = "v0.0.0-dev"
+
 func main() {
 	fs := config.BuildFlagSet()
+	if err := fs.Parse(os.Args[1:]); err != nil {
+		panic(fmt.Errorf("couldn't parse flags: %w", err))
+	}
+	handleInfoDisplayFlags(fs)
```

Comment:
Can you add a comment explaining that we check for the version key, and if it's found we display it and exit?